Sources/web3swift/Tokens/ERC1155/Web3+ERC1155.swift

```diff
@@ -137,7 +137,9 @@ public class ERC1155: IERC1155 {
     public func supportsInterface(interfaceID: String) async throws -> Bool {
         let contract = self.contract
         self.transaction.callOnBlock = .latest
-        self.transaction.gasLimitPolicy = .manual(30000)
+        // FIXME: check if this still works
+        self.transaction.gasLimit = 300000
+//        self.transaction.gasLimitPolicy = .manual(30000)
```

Comment:
So about these (and the ones below). I think I can just remove the gas limit all together here as those are read operations/transactions and setting a gas limit should have no effect.

I checked that  `supportsInterface` is definitely a `view` function too: https://docs.openzeppelin.com/contracts/4.x/api/utils#ERC165

Reply:
It's up to you, I'm quite dunno in terms of tx gas management.

Reply:
I checked that some of the newer written functions of `supportsInterface` also don't use `gasLimit`. So should be fine. 👍